source/_components/splunk.markdown

```diff
@@ -8,16 +8,15 @@ comments: false
 sharing: true
 footer: true
 logo: splunk.png
-ha_category:
-  - "History"
+ha_category: "History"
 ha_release: 0.13
 ---
 
-The `splunk` integration makes it possible to log all state changes to an external [Splunk](http://splunk.com/) database using Splunk's HTTP Event Collector (HEC) feature. You can either use this alone, or with the Home Assistant for Splunk [app](https://github.com/miniconfig/splunk-homeassistant). Since the HEC feature is new to Splunk, you will need to use at least version 6.3.
+The `splunk` component makes it possible to log all state changes to an external [Splunk](http://splunk.com/) database using Splunk's HTTP Event Collector (HEC) feature. You can either use this alone, or with the Home Assistant for Splunk [app](https://github.com/miniconfig/splunk-homeassistant). Since the HEC feature is new to Splunk, you will need to use at least version 6.3.
 
-## Configuration
+## {% linkable_title Configuration %}
 
-To use the `splunk` integration in your installation, add the following to your `configuration.yaml` file:
+To use the `splunk` component in your installation, add the following to your `configuration.yaml` file:
```

Comment:
```suggestion
To use the `splunk` integration in your installation, add the following to your `configuration.yaml` file:
```

Reply:
@klaasnicolaas - All the changes you've suggested are outside of the scope of my PR and my changes.  Is there a reason to make all these changes piggy-backed on this PR as opposed to it's own "cleanup" PR?

Reply:
Oh.  Maybe this isn't merged.  I confused the "parent-merged" tag.  So, all these suggestions... they aren't related to the changes being made.  Shouldn't they all be in a separate PR for general cleanup.  I don't care if it's accepted as part of this change, but it isn't a very clean association to have a bunch of changes unrelated to the stated purpose of the PR.  Lemme know.  I can accept all the suggestions.  Just wanted to raise the question and understand the approach used here.  Thanks.

Reply:
We have recently made major improvements to the docs website, which means that we thoroughly check all PRs whether inside or outside the scope and immediately align them with how we want to see it.

In this case, in addition to your added `filter` config variable, there are a few changes in your PR that undo changes (from our general cleanup) that have been made. Through the suggestions it's again in line with how we want to see it 😉

Reply:
Ah. Okay. I’ll apply the suggestions.

Reply:
Resolved.